rotkehlchen/constants/assets.py

```diff
@@ -260,3 +260,5 @@
 A_ALETH = Asset('eip155:1/erc20:0x0100546F2cD4C9D97f798fFC9755E47865FF7Ee6')
 A_STETH = Asset('eip155:1/erc20:0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84')
 A_SAND = Asset('eip155:1/erc20:0x3845badAde8e6dFF049820680d1F14bD3903a5d0')
+
+CONSTANT_ASSETS = {var_data for var_name, var_data in locals().items() if var_name.startswith('A_')}  # noqa: E501
```

Comment:
why do you need the if? It's all the locals here. Could also have reintrodued the create constants assets script but I guess it's not needed and would be an overkill

Reply:
the `if` is needed because locals have other things in here like `__name__`, `__doc__`, `__package__`...